lib/active_merchant/billing/credit_card.rb

```diff
@@ -176,21 +176,20 @@ def requires_verification_value?
       # @return [String]
       attr_accessor :icc_data
 
-      # Returns or sets a fallback reason for a EMV transaction whereby the customer's card entered a fallback scenario.
-      # This can be an arbitrary string.
+      # Returns or sets information about the source of the card data.
       #
       # @return [String]
-      attr_accessor :fallback_reason
-
-      # Returns or sets whether card-present EMV data has been read contactlessly.
-      #
-      # @return [true, false]
-      attr_accessor :contactless_emv
-
-      # Returns or sets whether card-present magstripe data has been read contactlessly.
-      #
-      # @return [true, false]
-      attr_accessor :contactless_magstripe
+      attr_accessor :read_method
+
+      READ_METHOD_DESCRIPTIONS = {
+        nil => 'A card reader was not used.',
+        'fallback_no_chip' => 'Magstripe was read because the card has no chip.',
+        'fallback_chip_error' => 'Magstripe was read because the card\'s chip failed.',
```

Comment:
Can do double quotes instead of escaping `'`: 
`"Magstripe was read because the card's chip failed."`